src/textbooks/Textbooks.scss

```diff
@@ -1,3 +1,8 @@
+@import "./scss/variables";
 @import "./empty-placeholder/EmptyPlaceholder";
 @import "./textbook-card/TextbookCard";
 @import "./textbook-form/TextbookForm";
+
+.alert-toast {
+  z-index: $popover !important;
```

Comment:
[important]: Try to use Paragon SCSS variable

```suggestion
  z-index: $zindex-tooltip !important;
```

Reply:
@PKulkoRaccoonGang We have the same variable for certificates page, should we change there also?

Reply:
i remember that you suggested exactly this naming so im curious

Reply:
Fixed